.github/workflows/go-tests.yml

```diff
@@ -0,0 +1,20 @@
+on:
+  push:
+    branches:
+      - "feature/**"
+      - "bugfix/**"
+      - "hotfix/**"
+name: Golang Tests
+jobs:
```

Comment:
might be a bit cleaner to do something like this:

```
jobs:
  build:
    runs-on: ubuntu-latest
    strategy:
      matrix:
        go: ['1.14.1', '1.13.9', '1.12' , '1.11']
    name: Go ${{ matrix.go }} Test
    steps:
      - uses: actions/checkout@v2
      - name: Setup go
        uses: actions/setup-go@v1
        with:
          go-version: ${{ matrix.go }}
      - run: go test ./...
```

Reply:
will test multiple versions then 👍

Reply:
Ahh nice, good idea